pages/app-layout/with-drawers.page.tsx

```diff
@@ -0,0 +1,151 @@
+                iconName: 'security',
+              },
+            },
+            {
+              ariaLabels: {
+                closeButton: 'ProHelp close button',
+                content: 'ProHelp drawer content',
+                triggerButton: 'ProHelp trigger button',
+                resizeHandle: 'ProHelp resize handle',
+              },
+              content: <ProHelp />,
+              id: 'pro-help',
+              trigger: {
+                iconSvg: <IconBriefcase />,
+              },
+            },
+          ],
+          onChange: (event: NonCancelableCustomEvent<string>) => setActiveDrawerId(event.detail),
+        },
+      };
+
+  return (
+    <ScreenshotArea gutters={false}>
+      <AppLayout
+        ariaLabels={appLayoutLabels}
+        breadcrumbs={<Breadcrumbs />}
+        content={
+          <ContentLayout
+            header={
+              <SpaceBetween size="m">
+                <Header variant="h1" description="Sometimes you need custom drawers to get the job done.">
+                  Testing Custom Drawers!
+                </Header>
+
+                <SpaceBetween size="xs">
+                  <Toggle checked={useTools} onChange={({ detail }) => setUseTools(detail.checked)}>
+                    Use Tools
+                  </Toggle>
+
+                  <Toggle checked={useDrawers} onChange={({ detail }) => setUseDrawers(detail.checked)}>
+                    Use Drawers
+                  </Toggle>
+                </SpaceBetween>
+              </SpaceBetween>
+            }
+          >
+            <Containers />
+          </ContentLayout>
+        }
+        contentType="form"
+        maxContentWidth={999999999999999}
+        splitPanel={
+          <SplitPanel
+            header="Split panel header"
+            i18nStrings={{
+              preferencesTitle: 'Preferences',
+              preferencesPositionLabel: 'Split panel position',
+              preferencesPositionDescription: 'Choose the default split panel position for the service.',
+              preferencesPositionSide: 'Side',
+              preferencesPositionBottom: 'Bottom',
+              preferencesConfirm: 'Confirm',
+              preferencesCancel: 'Cancel',
+              closeButtonAriaLabel: 'Close panel',
+              openButtonAriaLabel: 'Open panel',
+              resizeHandleAriaLabel: 'Slider',
+            }}
+          >
+            This is the Split Panel!
+          </SplitPanel>
+        }
+        onToolsChange={event => {
+          setIsToolsOpen(event.detail.open);
+        }}
+        tools={<Info />}
+        toolsOpen={isToolsOpen}
+        toolsHide={!useTools}
+        {...drawers}
+      />
+    </ScreenshotArea>
+  );
+}
+
+function IconBriefcase() {
+  return (
+    <svg viewBox="0 0 16 16" fill="none">
+      <path
+        d="M8 15H1V5h14v3m-6.5 4.25H16M12.25 8.5V16M5 1h6v4H5V1Z"
+        stroke="#fff"
+        strokeWidth="2"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+}
```

Comment:
I don't have a super strong opinion here, but shouldn't our testing pages test as many versions of the component as possible? It could be possible there are small alignment or other issues. I can remove for now.

Reply:
In other testing pages we use more abstract shapes for custom icons, for example in token group: https://github.com/cloudscape-design/components/blob/7ed751a10e2ecf63cec865877bf8a6f75b6f1e54/pages/token-group/permutations.page.tsx#L152-L174

I think, it is important to not accidentally create an unofficial icon set and this is why generic shapes are recommended